src/jtag/drivers/bcm2835gpio.c

```diff
@@ -530,8 +550,8 @@ static int bcm2835gpio_quit(void)
 	restore_gpio(ADAPTER_GPIO_IDX_SRST);
 	restore_gpio(ADAPTER_GPIO_IDX_LED);
 
-	/* Restore drive strength. MSB is password ("5A") */
-	pads_base[BCM2835_PADS_GPIO_0_27_OFFSET] = 0x5A000000 | initial_drive_strength_etc;
+    for (unsigned int i = 0; i < sizeof(initial_drive_strengths) / sizeof(initial_drive_strengths[0]); i++)
+		pads_base[BCM2835_PADS_GPIO_0_27_OFFSET + i] = BCM2835_PADS_GPIO_PASSWORD | initial_drive_strengths[i];
```

Comment:
The alignment of this function looks wrong, at least in the diff, which causes it to visually look like `bcm2835gpio_munmap` is getting called for each iteration of the loop.

This isn't the case, but we should probably unpack this loop anyway. given the above comment